### PR TITLE
add ventureId to interfaces and api calls

### PR DESCRIPTION
**Pull request by marcusellison**

### CODE REVIEW

File: src/module/api/audience/create/index.ts [hunk 1]
```diff
@@ -27,6 +34,7 @@ export async function Create(
   objProperty.setUserList(userList); // Right now an audience doesn't have to have members
   obj.getMetadataMap().set(key.UserID, userId);
   obj.getMetadataMap().set(key.OrganizationID, organizationId);
+  obj.getMetadataMap().set(key.OrganizationID, ventureId);
```
Comment (xh3b4sd) [inline src/module/api/audience/create/index.ts:37]: I think this should be the venture ID key.

File: src/component/sidebar/index.tsx [hunk 2]
```diff
@@ -40,6 +40,7 @@ function Sidebar(props: SidebarProps) {
   const timelineSearch: ITimelineQuery = {
     userId,
     organizationId,
+    ventureId: organizationId,
```
Comment (xh3b4sd) [inline src/component/sidebar/index.tsx:43]: My understanding is that this is our own code in the webclient only. So we could actually just get it straight right away. The only relevant part for migration is the metadata which goes over network.

File: src/module/api/texupd/create/index.ts [hunk 3]
```diff
@@ -20,6 +20,7 @@ export async function Create(newUpdate: INewUpdate): Promise<any> {
 
   objProperty.setText(newUpdate.text);
   obj.getMetadataMap().set(key.OrganizationID, newUpdate.organizationId);
+  obj.getMetadataMap().set(key.ventureID, newUpdate.ventureId);
```
Comment (xh3b4sd) [inline src/module/api/texupd/create/index.ts:23]: Is there a reason why this is now lowercase and all others are upper case?

Comment (marcusellison) [reply src/module/api/texupd/create/index.ts:23]: nope - just fixed consistency

**Pull request merged**